domain-models-runtime/src/test/java/org/folio/rest/persist/PostgresClientIT.java

```diff
@@ -1497,6 +1501,8 @@ public void streamGetDistinctOn(TestContext context) throws IOException {
     async.awaitSuccess();
   }
 
+  // While facets are passed, this does NOT seem to deal with facets
+  // in fact it, quite possibly streamGet do not support facets at all
```

Comment:
Can you fix the wording?

Reply:
I don't know how. I don't think streamGet even deal with facets.. Is that something I should investigate? (There's no regression.-- streamGet possibly never supported facts)